heron/spi/src/java/com/twitter/heron/spi/common/ShellUtils.java

```diff
@@ -92,10 +89,12 @@ public static int runSyncProcess(
       }
       pStdOut.append(inputstreamToString(process.getInputStream()));
       pStdErr.append(inputstreamToString(process.getErrorStream()));
-      if (verbose) {
-        LOG.info(pStdOut.toString());
-        LOG.info(pStdErr.toString());
-      }
+
+      LOG.log(Level.FINE, "\tSTDOUT: {0}", pStdOut.toString());
```

Comment:
Seems like Level.FINE logging should only be done when a verbose flag is passed. If there's general info logs that we want to show, could we instead change those to Level.INFO, instead of logging all of Level.FINE?